Update jiramicrosoft-tutorial.md (#1218)
MicrosoftDocs/entra-docs: docs/identity/saas-apps/jiramicrosoft-tutorial.md

## Supported versions of JIRA

* JIRA Core and Software: 7.0 to 9.17.4.
* JIRA Service Desk 3.0 to 4.22.1.
* JIRA also supports 5.2. For more details, click [Microsoft Entra single sign-on for JIRA 5.2](jira52microsoft-tutorial.md).

> [!NOTE]
> Please note that our JIRA Plugin also works on Ubuntu Version 16.04 and Linux.

> ## Microsoft SSO Plugins Market Place Data center and Server Side Application Link

Reviewer: ```suggestion
## Microsoft SSO plugins
```

> * [Microsoft Azure Active Directory single sign-on for JIRA Data-Center Application](https://marketplace.atlassian.com/apps/1224430/microsoft-azure-active-directory-single-sign-on-for-jira?tab=overview&hosting=datacenter
> )
> 
> * [Microsoft Azure Active Directory single sign-on for JIRA Server side Application](https://www.microsoft.com/en-us/download/details.aspx?id=56506)

Reviewer: ```suggestion
* [Microsoft Entra ID single sign-on for JIRA datacenter application](https://marketplace.atlassian.com/apps/1224430/microsoft-azure-active-directory-single-sign-on-for-jira?tab=overview&hosting=datacenter)

* [Microsoft Entra ID single sign-on for JIRA server-side application](https://www.microsoft.com/en-us/download/details.aspx?id=56506)
```
Azure Active Directory has been rebranded to Microsoft Entra ID.

	
	d. In **Login Button Description** type the description of button your organization wants the users to see on login screen.

> 	e.  In **Default Group** Select your organization Default Group to assign to new users (Default groups facilitate organized access rights to new user account).

Reviewer: ```suggestion
	e.  In **Default Group**, select your organization default group to assign to new users. Default groups facilitate organized access rights to new user accounts.
```

	f. In **SAML User ID Locations** select either **User ID is in the NameIdentifier element of the Subject statement** or **User ID is in an Attribute element**.  This ID has to be the JIRA user ID. If the user ID is not matched, then the system doesn't allow users to sign in.

	> [!Note]
	> Default SAML User ID location is Name Identifier. You can change this to an attribute option and enter the appropriate attribute name.

> 	g. If you select **User ID is in an Attribute element** option, then in **Attribute name** textbox type the name of the attribute where User ID is expected.

Reviewer: ```suggestion
	g. If you select **User ID is in an Attribute element**, then for **Attribute name**, type the name of the attribute where the user ID is expected.
```

> 	g. If you select **User ID is in an Attribute element** option, then in **Attribute name** textbox type the name of the attribute where User ID is expected.
> 
> 	h.  In **Auto-create User** feature (JIT User Provisioning): It automates user account creation in authorized web applications, without the need for manual provisioning. This reduces administrative workload and increases productivity. Because JIT relies on the login response from Azure AD, enter the SAML-response attribute values, which include the user's email address, last name, and first name.

Reviewer: ```suggestion
	g. If you select **User ID is in an Attribute element**, then for **Attribute name**, type the name of the attribute where the user ID is expected.

	h.  The **Auto-create User** feature (JIT User Provisioning) automates user account creation in authorized web applications, without the need for manual provisioning. This reduces administrative workload and increases productivity. Because JIT relies on the login response from Microsoft Entra ID, enter the SAML-response attribute values, including the user's email address, last name, and first name.
```

	i. If you are using the federated domain (like ADFS, and so on) with Microsoft Entra ID, then click on the **Enable Home Realm Discovery** option and configure the **Domain Name**.

	j. In **Domain Name** type the domain name here in case of the ADFS-based login.

> 	k. Check **Enable Single Sign out** if you wish to sign out from Microsoft Entra ID when a user sign out from JIRA.

Reviewer: ```suggestion
	k. Select **Enable Single Sign out** if you want to sign out of Microsoft Entra ID when a user signs out of JIRA.
```

> 	l. Enable **Force Azure Login** checkbox, if you wish to sign in through Microsoft Entra credentials only.

Reviewer: ```suggestion
	l. Enable **Force Azure Login** if you want to sign in by using only Microsoft Entra ID credentials.
```

	
	> [!Note]
	> To enable the default login form for admin login on login page when force azure login is enabled, add the query parameter in the browser URL.
	> `https://<domain:port>/login.jsp?force_azure_login=false`

	m. **Enable Use of Application Proxy** checkbox, if you have configured your on-premises atlassian application in an application proxy setup.

	* For App proxy setup , follow the steps on the [Microsoft Entra application proxy Documentation](~/identity/app-proxy/overview-what-is-app-proxy.md).

> 	n. Click **Save** button to save the settings.

Reviewer: ```suggestion
	n. Select **Save** to save the settings.
```
